### PR TITLE
Qcar 25

### PR DESCRIPTION
**Pull request by asimha**

### CODE REVIEW

File: app/models/qcareer_configuration.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class QcareerConfiguration < ActiveRecord::Base
+
+  validates :value, presence: true
```
Comment (kpvarma) [inline app/models/qcareer_configuration.rb]: name is also mandatory.

Comment (kpvarma) [inline app/models/qcareer_configuration.rb]: name should be one in the list [HOMEPAGE_EVENT, XXXX]

Comment (asimha) [inline app/models/qcareer_configuration.rb]: yes, But we are not editing name in configuration. we are just displaying, so I didn't give validation for that

Comment (kpvarma) [inline app/models/qcareer_configuration.rb]: I agree. But still to make it perfect, add a validation.

**Comment by kpvarma** [comment]
Any more questions on this ?

**Comment by asimha** [comment]
No question on this

File: app/models/qcareer_configuration.rb [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+  validates :description, presence: true
+  validates :name, presence: true
```
Comment (asimha) [inline app/models/qcareer_configuration.rb]: Here i have added the validation for name

File: spec/models/qcareer_configuration_spec.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+  it "should not validate the career configuation fields if any of the field is nill " do
+    @career_configuration.value = nil
+    expect(@career_configuration).not_to be_valid
+    
+    @career_configuration.description = nil
+    expect(@career_configuration).not_to be_valid
+  end
+
+  it "should not validate if the name is nill" do
```
Comment (asimha) [inline spec/models/qcareer_configuration_spec.rb]: test cases for name validation

**Pull request merged**